src/styles/_GameCard.scss

```diff
@@ -2,7 +2,7 @@
   background: #FFF;
   border: outset 4px #2b3034;
   color:#fff;
-  height: 240px;
+  height: 250px;
```

Comment:
Made the cards a little taller so that the game photo didn't sit right at the bottom of the box.